include/alpaka/core/ConcurrentExecPool.hpp

```diff
@@ -174,7 +172,7 @@ namespace alpaka
 
             private:
                 //! The execution function.
-                virtual auto run() -> void final
```

Comment:
Why is the `virtual` removed here?

Reply:
`override` and `final` imply `virtual`, so `clang-tidy` removes it

Reply:
Ah, I see. Thanks!